.github/workflows/golangci-lint.yml

```diff
@@ -18,16 +18,20 @@ jobs:
   golangci:
     name: lint
     runs-on: ubuntu-24.04
+    strategy:
+      matrix:
+        dirs: [".", "aks-node-controller"]
```

Comment:
which Folder were subject to the linter before ?  should we add more folders ?

Reply:
yep

Reply:
Another option is to find all `go.mod` files and run linters on them.

But there is a few ad-hoc scripts and e2e I'm not sure I want to bother with lint errors.
Maybe explicitly adding important ones is cleaner. Easier to maintain the pipieline.

I don't think we have many other folders we worry about.